prow/spyglass/viewers/buildlog/buildlog_viewer.go

```diff
@@ -258,16 +247,78 @@ func logLinesAll(artifact viewers.Artifact) []string {
 	if err != nil {
 		if err == viewers.ErrFileTooLarge {
 			logrus.WithError(err).Error("Artifact too large to read all.")
-			return []string{}
+		} else {
+			logrus.WithError(err).Error("Failed to read log.")
```

Comment:
Should consider displaying this to the user somehow instead of logging, again perhaps OOS.

Reply:
Agreed. logLinesAll should probably return an error. I'll do this in a separate PR.

Reply:
please file an issue for follow-up and I'm happy to get this merged! looking forward to build log improvements :-)

Reply:
#9482